pages/index.js

```diff
@@ -185,6 +185,50 @@ export default function Home() {
       </div>
       </Hero2>
 
+      <Hero3>
+        <div className='in-container'>
+          <div className='h2-partners'> Partners </div>
+          <div className='eight-posts'>
+            <div className='home-card'>
+                <a href="https://www.tii.ae/quantum"> 
+                  <img src='/images/home/tii.png' alt='/'/> 
+                </a>
+            </div>
+            <div className='home-card'>
+                <a href="https://home.infn.it/it/"> 
+                  <img src='/images/home/infn.png' alt='/'/> 
+                </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://home.cern/"> 
+                <img src='/images/home/cern.png' alt='/'/> 
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.nvidia.com/en-usqilimanjaro
+              "> 
+                <img src='/images/home/nvidia.png' alt='/'/>
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.qilimanjaro.tech/"> 
+                <img src='/images/home/qilimanjaro.png' alt='/'/> 
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.a-star.edu.sg/ihpc"> 
+                <img src='/images/home/ihpc.png' alt='/'/> 
+              </a>
+            </div>
+            <div className='home-card'>
+              <a href="https://www.quantumlah.org/"> 
+                <img src='/images/home/cqt.png' alt='/'/>
+              </a>
+            </div>
+          </div>
+        </div>
+      </Hero3>
```

Comment:
I would limit the `Hero` to a single one, and the rest of the page to a more ordinary flow.
I wonder if using a `Hero` at all is really useful. However, for this round fine as is.

Reply:
Thank you. I open an issue for this.